src/base/bittorrent/magneturi.cpp

```diff
@@ -80,7 +80,7 @@ MagnetUri::MagnetUri(const QString &source)
     if (source.isEmpty()) return;
 
     if (isV2Hash(source))
-        m_url = QString::fromLatin1("magnet:?xt=urn:btmh:") + source;
+        m_url = QString::fromLatin1("magnet:?xt=urn:btmh:1220") + source; //0x12 0x20 is the "multi-hash format" tag for the SHA-256 hashing scheme.
```

Comment:
please add space after `//`.

Reply:
after reading [this](https://github.com/multiformats/multihash), I think you should rename "multi-hash" to "multihash".

Reply:
done